index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Racha's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link 
+      href="https://fonts.googleapis.com/css?family=Poppins:100,200,300,400,500,700"  
+      rel="stylesheet">
+</head>
+<body>
+    <header >
+        <nav class="nav-bar">
+            <ul>
+              <li>
+                <a class="logo" href="index.html">Racha</a>
+              </li>
+            </ul>
+            <ul class="links-holder">
+              <li class="link"><a href="index.html">Portoflio</a></li>
+              <li class="link"><a href="#">About</a></li>
+              <li class="link"><a href="#">Contact</a></li>
+            </ul>
+            <div class="hamburger">
+              <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  fill="currentColor"
+                  class="bi bi-list"
+                  viewBox="0 0 16 16"
+                >
+                  <path
+                    fill-rule="evenodd"
+                    d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"
+                  />
+                </svg>
+            </div>
+          </nav>
+        
+    </header>
+    <main>
+      <!--Headline section-->
+      <section class="headline-section">
```

Comment:
- [ ] Great job adding fonts and font sizes to match the design. As requested by the previous reviewer, kindly fix all spacings too to match the design template. You are expected to stick to design as much as possible.
### Your design
![image](https://user-images.githubusercontent.com/94551484/227311924-0aa9f993-c8fb-4149-a5f2-45eb06d9b3a1.png)
### Figma template
![image](https://user-images.githubusercontent.com/94551484/227312070-05ee8e51-cb96-4450-8e38-dd650638713a.png)